config/chassis_modules.py

```diff
@@ -45,5 +74,13 @@ def shutdown_chassis_module(db, chassis_module_name):
 def startup_chassis_module(db, chassis_module_name):
     """Chassis-module startup of module"""
     config_db = db.cfgdb
+    ctx = click.get_current_context()
+    
+    #To avoid duplicate operation
+    if get_config_module_state(db, chassis_module_name) is None:
+        click.echo("Duplicate operation for " + chassis_module_name)
+        return
```

Comment:
It should not be "none".  Also, should log "Module {} is ready set to up state".